adage/analyze/tests.py

```diff
@@ -907,3 +928,100 @@ def test_experiment_filter_in_sample(self):
 
         # Test non-get methods too.
         self.call_non_get_API(filter_uri, data=data)
+
+    def create_expressionvalue_data(self, num_genes):
+        """
+        Generate test data for ExpressionValueResource
+        """
+        if Organism.objects.exists():
+            organism = Organism.objects.first()
+        else:
+            organism = factory.create(Organism)
+
+        if MLModel.objects.exists():
+            ml_model = MLModel.objects.first()
+        else:
+            ml_model = factory.create(MLModel)
+
+        # Create genes:
+        for i in range(num_genes):
+            Gene.objects.create(entrezid=(i + 1),
+                                systematic_name="sys_name #" + str(i + 1),
+                                standard_name="sys_name #" + str(i + 1),
+                                organism=organism)
+
+        # Note: no need to create samples. See setUp() above.
+
+        # Now, for each sample, randomly select Genes and create
+        # ExpressionValue objects for them
+        self.expressionvalue_total = 0
```

Comment:
The variable name `expressionvalue_total` is a little misleading. (I originally thought it was the sum of expression values instead of the number of expression value records in the database). Maybe we can change it into  "expression_value_counter" or something like that?